pkg/capture/buffer.go

```diff
@@ -4,16 +4,16 @@ import (
 	"unsafe"
 
 	"github.com/els0r/goProbe/cmd/goProbe/config"
+	"github.com/els0r/goProbe/pkg/capture/capturetypes"
 	"github.com/fako1024/gotools/concurrency"
-	"github.com/fako1024/slimcap/capture"
 	"golang.org/x/sys/unix"
 )
 
 const (
 
-	// bufElementAddSize denotes the required fixed size component for the pktType and pktSize
-	// values per element: 1 (pktType) + 4 (uint32)
-	bufElementAddSize = 5
+	// bufElementAddSize denotes the required size for a buffer element
+	// (size of EPHash + 4 bytes for pktSize + 1 byte for pktType, isIPv4, auxInfo, errno, respectively)
+	bufElementSize = capturetypes.EPHashSize + 8
```

Comment:
Idea as discussed: we instrument the part with a gauge, measuring how big the buffer was right before it was flushed. This will provide an indicator of how much of the configured buffer space was used.